README.md

```diff
@@ -85,9 +81,20 @@ Almost the same as the original VSCode extension, with a few additional settings
 
 See the configuration schema [here](./packages/service/configuration.schema.json).
 
-## Known Issues
+## Troubleshooting
+
+Please see [Known issues](https://github.com/yioneko/vtsls/issues/26) first.
+
+### Bad performance of completion
+
+`tsserver` could throw out plenty of completion entries, most of them are globally accessible variables, modules or namespaces. Some LSP clients have poor performance for fuzzy matching or filtering, and cause noticeable delay of completion.
+
+Instead of switching client, some server configuration options could also make partial optimizations:
 
-See [Known issues](https://github.com/yioneko/vtsls/issues/26)
+- `vtsls.experimental.completion.enableServerSideFuzzyMatch`: before returning all the completion candidates from `tsserver`, the server will do fuzzy matching and filter out entries with no match. This can significantly reduce the number of invalid entries.
```

Comment:
Yes, the option alone has very limited effect on filtering. A better description is needed.